cypress/fixtures/successful_sign_up.json

```diff
@@ -0,0 +1,11 @@
+{
+  "status": "success",
+  "data": {
+    "id": 1,
+    "email": "user@mail.com",
+    "provider": "email",
+    "uid": "user@mail.com",
+    "name": "User",
```

Comment:
Will you get name from the api?

Reply:
Team API informs me that we will not, good catch.  Removed.